src/inverse/cholesky/gpu.cpp

```diff
@@ -0,0 +1,25 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <complex>
+
+#include <dlaf/inverse/cholesky/impl.h>
+
+namespace dlaf {
+namespace inverse {
+namespace internal {
+
+DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, float)
+DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, double)
+DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, std::complex<float>)
+DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, std::complex<double>)
+}
+}
+}
```

Comment:
```suggestion
namespace dlaf::inverse::internal {

DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, float)
DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, double)
DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, std::complex<float>)
DLAF_INVERSE_CHOLESKY_ETI(, Backend::GPU, Device::GPU, std::complex<double>)
}
```
In other files as well.